doc/develop/plugins.rst

```diff
@@ -278,6 +278,7 @@ When that entry is selected, it shows an acquisition window and then acquire
             ccd = self.main_data.ccd
             exp = self.exposureTime.value
             ccd.exposureTime.value = exp
+            dlg.pauseSettings()	# Freezes the setting and stream controls in window
```

Comment:
You need to also _document_ these new methods (after documentation `setAcquisitionInfo()`).